File: ejercicios/python/ejercicio009.py
from math import *

a = int(input("Ingrese el primer nœmero: "))
b = int(input("Ingrese el segundo nœmero: "))
if a > 0 and b > 0:
    print("la suma de " + str(a) + " + " + str(b) + " es " + str(a+b))
    print("la resta de " + str(a) + " - " + str(b) + " es " + str(a-b))
    print("el producto de " + str(a) + " * " + str(b) + " es " + str(a*b))
    print("la divisi—n de " + str(a) + " / " + str(b) + " es " + str(a//b))
else:
    print("Error en el ingreso de los datos " + a + " o " + b)

File: ejercicios/c++/ejercicio009.cpp
#include<iostream>

using namespace std;

int main() {

	int a;
	int b;

	cout<<"Ingrese el primer nœmero: ";
	cin>>a;
	cout<<"Ingrese el segundo nœmero: ";
	cin>>b;
	if (a>0 && b>0) {
		cout<<"la suma de "<<a<<" + "<<b<<" es "<<a+b<<endl;
		cout<<"la resta de "<<a<<" - "<<b<<" es "<<a-b<<endl;
		cout<<"el producto de "<<a<<" * "<<b<<" es "<<a*b<<endl;
		cout<<"la division de "<<a<<" / "<<b<<" es "<<a/b<<endl;
	} else {
		cout<<"Error en el ingreso de los datos "<<a<<" o "<<b<<endl;
	}

	return 0;
}

